modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -219,6 +219,11 @@ public void setUp() throws Exception {
 					0, ObjectFieldConstants.BUSINESS_TYPE_TEXT, null,
 					ObjectFieldConstants.DB_TYPE_STRING, true, false, null,
 					"Last Name", "lastName", false, true)));
+
+		_objectDefinition.setName("A" + RandomTestUtil.randomString());
+
+		_objectDefinitionLocalService.updateObjectDefinition(_objectDefinition);
+
```

Comment:
@icgsbr I don't understand this change. 

The `ObjectDefinitionTestUtil.addCustomObjectDefinition` is already generating a random name with the same pattern: see [here](https://github.com/carolmariaabb/liferay-portal/blob/368c8f1740915dd0a022a092aed686ad5acccd06/modules/apps/object/object-test-util/src/main/java/com/liferay/object/test/util/ObjectDefinitionTestUtil.java#L42) and [here](https://github.com/carolmariaabb/liferay-portal/blob/368c8f1740915dd0a022a092aed686ad5acccd06/modules/apps/object/object-test-util/src/main/java/com/liferay/object/test/util/ObjectDefinitionTestUtil.java#L143).